db/db_impl.cc

```diff
@@ -1153,6 +1154,28 @@ Status DBImpl::RecoverLogFiles(const std::vector<uint64_t>& log_numbers,
       }
       WriteBatchInternal::SetContents(&batch, record);
 
+#ifndef ROCKSDB_LITE
+      if (db_options_.wal_filter != nullptr) {
+        WALFilter::WALProcessingOption walProcessingOption =
```

Comment:
There are both of naming like "WALFilter" and "WalFilter" in the code base. We should consolidate it some time. "Wal" seems to be more to me so I would vote "WalFilter".